content/en/users/tutorials/filetransfer-gridstorage/_index.md

```diff
@@ -8,4 +8,229 @@ description: >
+
+## FTS
+
+### Step 1 Configuration check
+
+To verify that everything is configured properly you can check with the
+following command pointing to the cerficates directly:
+
+<!-- markdownlint-disable line-length -->
+
+```shell
+$ fts-rest-whoami --key .globus/userkey.pem --cert .globus/usercert.pem -s https://fts3-public.cern.ch:8446/
+User DN: /DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Andrea Cristofori ac@egi.eu
+VO: AndreaCristoforiac@egi.eu@tcs.terena.org
+VO id: XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
+Delegation id: XXXXXXXXXXXXXXXX
+Base id: XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
+```
+
+<!-- markdownlint-enable line-length -->
+
+In general the commands can be used by specifying the user public and private
+key like shown in the example or by creating a proxy certificate as
+
+### Proxy creation
+
+To avoid the need to specify at each command the private and public key if
+everything is already configured and working properly it will be enough to
+execute the following coomand:
+
+<!-- markdownlint-disable line-length -->
+
+```shell
+$ voms-proxy-init
+Your identity: /DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Andrea Cristofori ac@egi.eu
+Creating proxy ........................................... Done
+
+Your proxy is valid until Wed Aug 25 04:18:14 2021
+```
+
+<!-- markdownlint-enable line-length -->
+
+As the output of the command shows a proxy certificate has been generated that
+will be valid, by default, for 12 hours. This can be usually increased for
+example to 48 hours with the following option:
+
+<!-- markdownlint-disable line-length -->
+
+```shell
+$ voms-proxy-init -valid 48:00
+Your identity: /DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Andrea Cristofori ac@egi.eu
+Creating proxy ................................... Done
+
+Your proxy is valid until Thu Aug 26 16:23:01 2021
+```
+
+<!-- markdownlint-enable line-length -->
+
+To verify the timeleft for the validity of the proxy created use the following
+command:
+
+<!-- markdownlint-disable line-length -->
+
+```shell
+$ voms-proxy-info
+subject   : /DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Andrea Cristofori ac@egi.eu/CN=1451339003
+issuer    : /DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Andrea Cristofori ac@egi.eu
+identity  : /DC=org/DC=terena/DC=tcs/C=NL/O=Stichting EGI/CN=Andrea Cristofori ac@egi.eu
+type      : RFC compliant proxy
+strength  : 1024 bits
+path      : /tmp/x509up_u1000
+timeleft  : 19:59:57
+```
+
+<!-- markdownlint-enable line-length -->
+
+When the timeleft reaches zero if the previous command used to check the
+identity will return the following message:
+
+<!-- markdownlint-disable line-length -->
+
+```shell
+$ fts-rest-whoami -s https://fts3-public.cern.ch:8446/
+Error: Proxy expired!
+```
+
+<!-- markdownlint-enable line-length -->
+
+The last option that we need to use is specify the VO that we want to use for
+the proxy being created. In the following example the "dteam" VO has been used:
```

Comment:
Use `you` as previously done, not `we`.

Reply:
To be reviewed in the full text.

Reply:
Changed and changed in the text we with you.

Reply:
Thanks!